ui/src/app/cron-workflows/components/cron-workflow-details/cron-workflow-details.tsx

```diff
@@ -207,7 +225,42 @@ export const CronWorkflowDetails = ({match, location, history}: RouteComponentPr
                 <SlidingPanel isShown={!!sidePanel} onClose={() => setSidePanel(null)}>
                     {sidePanel === 'share' && <WidgetGallery namespace={namespace} label={'workflows.argoproj.io/cron-workflow=' + name} />}
                 </SlidingPanel>
+                <>
+                    <ErrorNotice error={error} />
+                    {!workflows ? (
+                        <ZeroState title='No completed cron-workflows'>
+                            <p> You can submit or resume cron workflow to run a cron-workflow. </p>
+                        </ZeroState>
+                    ) : (
+                        <div className='argo-table-list'>
+                            <div className='row argo-table-list__head'>
+                                <div className='columns small-1 workflows-list__status'></div>
+                                <div className='row small-11'>
+                                    <div className='columns small-2'>NAME</div>
+                                    <div className='columns small-1'>NAMESPACE</div>
+                                    <div className='columns small-1'>STARTED</div>
+                                    <div className='columns small-1'>FINISHED</div>
+                                    <div className='columns small-1'>DURATION</div>
+                                    <div className='columns small-1'>PROGRESS</div>
+                                    <div className='columns small-2'>MESSAGE</div>
+                                    <div className='columns small-1'>DETAILS</div>
+                                    <div className='columns small-1'>ARCHIVED</div>
+                                    {(columns || []).map(col => {
+                                        return (
+                                            <div className='columns small-1' key={col.key}>
+                                                {col.name}
+                                            </div>
+                                        );
+                                    })}
+                                </div>
+                            </div>
+                            {workflows.map(wf => {
```

Comment:
```suggestion
                            {/* checkboxes are not visible and are unused on this page */}
                            {workflows.map(wf => {
```

adding a comment so that it's clear to readers _why_ `null` is used

Reply:
Oh Adding a comment is important things!🫢